packages/cli/src/info/env_info.ts

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
What is the reason to not follow naming convention ?

if this is because we add types for output we don't controll then please move types to separate file.